student/src/main/java/tds/student/services/remote/RemoteOpportunityService.java

```diff
@@ -380,4 +402,70 @@ private static TestConfig mapExamConfigurationToTestConfig(ExamConfiguration exa
 
     return opportunitySegments;
   }
+
+  private List<TestSelection> mapAssessmentMetadataToTestSelections(final List<ExamAssessmentMetadata> assessmenMetadata,
+                                                                    final BrowserInfo browserInfo) throws ReturnStatusException {
+    List<TestSelection> selections = new ArrayList<>();
+
+    for (ExamAssessmentMetadata metadata : assessmenMetadata) {
+      TestSelection selection = new TestSelection();
+      selection.SetReturnStatus(new ReturnStatus(metadata.getStatus(), metadata.getDeniedReason()));
+      selection.setTestKey(metadata.getAssessmentKey());
+      selection.setTestID(metadata.getAssessmentId());
+      selection.setOpportunity(metadata.getAttempt());
+      selection.setMode("online");
+      selection.setDisplayName(metadata.getAssessmentLabel());
+      selection.setMaxOpportunities(metadata.getMaxAttempts());
+      selection.setSortOrder(0); // not used by UI
+      selection.setSubject(metadata.getSubject());
+      selection.setGrade(metadata.getGrade());
+      //TODO: Determine if we need this
+//      selection.getRequirements ().lookup (getTdsSettings ().getClientName (), testSelection);
```

Comment:
I'll remove the todo/commented out code - it does not appear that we need to make this call.